### PR TITLE
Added text on submit

### PR DESCRIPTION
**Pull request by jarrodyellets**
I added the text onSubmit.  Let me know if you need any changes.

### CODE REVIEW

File: src/components/Question.js [hunk 1]
```diff
@@ -19,10 +19,11 @@ const NavigationButton = props =>
 class Question extends Component {
   constructor() {
     super();
-    this.state = { userAnswerIndex: null, answerSubmitted: null, error: false };
+    this.state = { userAnswerIndex: null, answerSubmitted: null, error: false, correctAnswer: null };
   }
 
   handleSubmit(event) {
+    let currentAnswer = this.props.question.answers[this.state.userAnswerIndex];
```
Comment (austintackaberry) [inline src/components/Question.js]: I like that you determine the `currentAnswer` via props and can easily check if it is correct via `currentAnswer.isCorrect`.

File: src/components/Question.js [hunk 2]
```diff
@@ -40,6 +43,10 @@ class Question extends Component {
     this.setState({ userAnswerIndex: event.target.value });
   }
 
+  handleCorrectAnswer(answer) {
```
Comment (austintackaberry) [inline src/components/Question.js]: I think all of this logic could be handled in `handleSubmit`

File: src/components/Question.js [hunk 3]
```diff
@@ -119,19 +126,20 @@ class Question extends Component {
...
                 left: "0",
                 right: "0",
                 bottom: "2px",
-                fontSize: "14px"
+                fontSize: "18px",
+                fontWeight: "700"
               }}
             >
-              *Please Select an Answer*
+              {this.state.error && (<div>Please Select an Answer</div>)}
```
Comment (austintackaberry) [inline src/components/Question.js]: In order to make sure we only render one message, I think this message should be determined before the return statement in the render method. Something like this:

```javascript
let message;
if (this.state.error) {message = "Please select an answer"}
else if (this.state.corrrectAnswer) {message = "Correct!"}
else {message = "Incorrect"}

return (
  . . .
  <p>{message}</p>
  . . .
)
```

File: src/components/Question.js [hunk 4]
```diff
@@ -146,11 +154,13 @@ class Question extends Component {
             label="Previous"
             destination={navigation.previous.url}
             enabled={navigation.previous.enabled}
+            onClick={() => {this.handleCorrectAnswer(null)}}
```
Comment (austintackaberry) [inline src/components/Question.js]: Is this necessary? I would think that navigating to a new Question would reinitialize the component state which would automatically set the state to null.

Comment (jarrodyellets) [reply src/components/Question.js]: That makes sense, I'll switch that out.

File: src/components/Question.js (hunk 2 above)
Comment (jarrodyellets) [reply src/components/Question.js]: I agree, esp since its not being called by the nav button anymore.

**Comment by jarrodyellets** [comment]
Thanks for the suggestions, I'll get them implemented today.

**Comment by jarrodyellets** [comment]
Here is the update code, let me know what you think.

File: src/components/Question.js [hunk 5]
```diff
@@ -31,6 +32,8 @@ class Question extends Component {
       this.setState({
         answerSubmitted: true,
         error: false
+      }, () => {
+        currentAnswer.isCorrect ? this.setState({correctAnswer: true}) : this.setState({correctAnswer: false});
```
Comment (nikrb) [inline src/components/Question.js]: forgive my nuubness, but this doesn't look right, calling setState in the setState callback. Can we not just add correctAnswer into the initial call?
```
this.setState({
  correctAnswer: currentAnswer.isCorrect,
  answerSubmitted: true,
  error: false,
});
```

**Comment by jarrodyellets** [comment]
Updated the code per nik's suggestion.

**Pull request merged**